appengine/go11x/tasks/create_http_task/create_http_task_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"testing"
+)
+
+func TestCreateHttpTask(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	locationID := "us-central1"
+	queueID := "my-appengine-queue"
+	url := fmt.Sprintf("https://%s.appspot.com/task_handler", tc.ProjectID)
+
+	tests := []struct {
+		name    string
+		message string
+	}{
+		{
+			name:    "Message",
+			message: "task details for handler processing",
+		},
+		{
+			name:    "No Message",
+			message: "",
+		},
+	}
+
+	for _, test := range tests {
+		_, err := createHttpTask(tc.ProjectID, locationID, queueID, url, test.message)
```

Comment:
Any sort of cleanup we should do for the task?

Reply:
We can implement delete task to clean up the created task.

Reply:
This would be good as an additional sample (not just a test). I currently don't have time to implement.